### PR TITLE
Set phase name as required

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/8262

## Testing
**URL to test:** 
https://8262-blank-phase--atd-moped-main.netlify.app/moped/projects

**Steps to test:**
Try to add a phase to the timelines table without entering any information. It should prevent you from being able to save.

I also fixed some warnings/errors in the console. 

---
#### Ship list
- [ ] Code reviewed 
- [x] Product manager approved
- [x] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectTimeline.js [hunk 1]
```diff
@@ -322,6 +324,7 @@ const ProjectTimeline = ({ refetch: refetchSummary }) => {
       title: "Phase name",
       field: "phase_name",
       lookup: phaseNameLookup,
+      validate: row => !!row.phase_name,
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectView/ProjectTimeline.js:327]: dont let them save the row unless theres a phase.

**Review by mateoclarke — APPROVED** [review]
🚢 💨 ✨

**Review by frankhereford — APPROVED** [review]
Tested, reviewed - this looks great to me - thank you!

**Review by amenity — APPROVED** [review]
LGTM! ✨ 🙏

**Pull request merged**